src/emulator/functionsEmulator.ts

```diff
@@ -440,14 +444,8 @@ export class FunctionsEmulator implements EmulatorInstance {
 
   /**
    * When a user changes their code, we need to look for triggers defined in their updates sources.
-   * To do this, we spin up a "diagnostic" runtime invocation. In other words, we pretend we're
-   * going to invoke a cloud function in the emulator, but stop short of actually running a function.
-   * Instead, we set up the environment and catch a special "triggers-parsed" log from the runtime
-   * then exit out.
-   *
-   * A "diagnostic" FunctionsRuntimeBundle looks just like a normal bundle except triggerId == "".
    *
-   * TODO(abehaskins): Gracefully handle removal of deleted function definitions
+   * TODO(danielylee): Gracefully handle removal of deleted function definitions
```

Comment:
small brain: leaving TODO to ex employee
big brain: changing TODO to self
galaxy brain: filing bug, putting it in our backlog, and leaving it in the TODO assignment